packages/webapp/pages/onboarding.tsx

```diff
@@ -142,6 +143,9 @@ const seo: NextSeoProps = {
 };
 
 export function OnboardPage(): ReactElement {
+  const params = new URLSearchParams(window.location.search);
+  const emailParam = params.get('email');
+  const authTriggerParam = params.get('authTrigger');
```

Comment:
also probably better lowercase, even obscure like `atr` to avoid users touching it